app/services/common_platform/api/search_prosecution_case.rb

```diff
@@ -7,14 +7,65 @@ class SearchProsecutionCase < ApplicationService
 
       def initialize(params)
         @response = ProsecutionCaseSearcher.call(**params)
+        @blank_defendants = []
       end
 
       def call
         check_response_status
-
+        check_for_blank_defendants(prosecution_cases)
         record_prosecution_cases
       end
 
+      def check_for_blank_defendants(prosecution_cases)
```

Comment:
I'd name this logBlankDefendants or something because that's what it's ultimately doing rather than like a check (returning a boolean).

Reply:
I've opted to keep this one as-is, as within this method I already have a call to an existing method: log_error_for_blank_defendants, which I think covers it.